tests/pr_test.go

```diff
@@ -37,55 +37,60 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestRunComplete(t *testing.T) {
+func TestRunFSCloudExample(t *testing.T) {
 	t.Parallel()
-
-	// Generate a 10 char long random string for the admin_pass
-	randomBytes := make([]byte, 10)
-	_, err := rand.Read(randomBytes)
-	// add character prefix to avoid generated password beginning with special char
-	randomPass := "A" + base64.URLEncoding.EncodeToString(randomBytes)[:10]
-
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:            t,
-		TerraformDir:       completeExampleTerraformDir,
-		Prefix:             "rabbitmq-cmp",
-		ResourceGroup:      resourceGroup,
-		BestRegionYAMLPath: regionSelectionPath,
+		Testing:      t,
+		TerraformDir: "examples/fscloud",
+		Prefix:       "rabbitmq-cmp",
+		Region:       "us-south", // For FSCloud locking into us-south since that is where the HPCS permanent instance is
+		/*
+		 Comment out the 'ResourceGroup' input to force this test to create a unique resource group to ensure tests do
+		 not clash. This is due to the fact that an auth policy may already exist in this resource group since we are
+		 re-using a permanent HPCS instance. By using a new resource group, the auth policy will not already exist
+		 since this module scopes auth policies by resource group.
+		*/
+		//ResourceGroup: resourceGroup,
 		TerraformVars: map[string]interface{}{
-			"rabbitmq_version": "3.11", // Always lock this test into the latest supported RabbitMQ version
-			"users": []map[string]interface{}{
-				{
-					"name":     "testuser",
-					"password": randomPass, // pragma: allowlist secret
-					"type":     "database",
-				},
-			},
-			"admin_pass": randomPass,
+			"access_tags":                permanentResources["accessTags"],
+			"existing_kms_instance_guid": permanentResources["hpcs_south"],
+			"kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+			"rabbitmq_version":           "3.11", // Always lock this test into the latest supported RabbitMQ version
 		},
 	})
+
 	options.SkipTestTearDown = true
 	output, err := options.RunTestConsistency()
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
-	outputs := terraform.OutputAll(options.Testing, options.TerraformOptions)
-	expectedOutputs := []string{"port", "hostname"}
-	_, outputErr := testhelper.ValidateTerraformOutputs(outputs, expectedOutputs...)
-	assert.NoErrorf(t, outputErr, "Some outputs not found or nil")
```

Comment:
since you have moved this assertion to the `TestRunCompleteUpgradeExample` test, you no longer need to call `SkipTestTearDown` and `TestTearDown`